test/unit/queue/src/QueueTest.cpp

```diff
@@ -113,7 +100,10 @@ TEMPLATE_LIST_TEST_CASE(
     }
 
     CHECK(callbackFinished.load() == true);
-    LOOPED_CHECK(30, 100, alpaka::empty(f.m_queue));
+
+    // The blocking queue must be empty because we wait for the non-blocking queue and the blocking is synchronized
+    // implicit.
```

Comment:
```suggestion
    // The blocking queue must be empty because we wait for the non-blocking queue and the blocking one is synchronized
    // implicitly.
```